app/assets/javascripts/leaflet.map.js

```diff
@@ -277,7 +278,19 @@ L.OSM.Map = L.Map.extend({
         headers: { accept: "application/json" },
         signal: this._objectLoader.signal
       })
-        .then(response => response.json())
+        .then(async response => {
+          if (response.ok) {
+            return response.json();
+          }
+
+          const status = response.statusText || response.status;
+          if (response.status !== 400 && response.status !== 509) {
```

Comment:
To ignore deleted element response errors:
```suggestion
          if (![400, 410, 509].includes(response.status)) {
```